lib/poparser/po.rb

```diff
@@ -36,6 +36,24 @@ def add(entry)
     end
     alias_method :<<, :add
 
+    # delete entry from po file
+    #
+    # @example
+    #   
+    #   delete(entry)
+    #
+    # @param entry  [PoParser::Entry] entry to delete
+    # @return [None]
+    def delete(entry)
+      puts entry.class.name
+      if entry.kind_of? PoParser::Entry
+        @entries.delete(entry)
+      else
+        raise ArgumentError, 'Must be an entry'
+      end
+      self
```

Comment:
It might be a good idea to return the deleted entry here to keep consistent with the way the ruby default data structures ([Array#delete](https://docs.ruby-lang.org/en/2.0.0/Array.html#method-i-delete ) and hash) are handling the delete command. 

```suggestion
     raise ArgumentError, 'Must be an entry' unless entry.kind_of? PoParser::Entry
     return @entries.delete(entry)
```
would be a nicer way to write this.